frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-page.tsx

```diff
@@ -71,12 +72,18 @@ export const OperatorHubList: React.FC<OperatorHubListProps> = ({
           const { currentCSVDesc } = (channels || []).find(({ name }) => name === defaultChannel);
           const currentCSVAnnotations: OperatorHubCSVAnnotations =
             currentCSVDesc?.annotations ?? {};
-          const [infraFeatures, validSubscription] = ANNOTATIONS_WITH_JSON.map((annotationKey) => {
-            return parseJSONAnnotation(currentCSVAnnotations, annotationKey, () =>
-              // eslint-disable-next-line no-console
-              console.warn(`Error parsing annotation in PackageManifest ${pkg.metadata.name}`),
-            );
-          });
+          const [parsedInfraFeatures = [], validSubscription] = ANNOTATIONS_WITH_JSON.map(
+            (annotationKey) => {
+              return parseJSONAnnotation(currentCSVAnnotations, annotationKey, () =>
+                // eslint-disable-next-line no-console
+                console.warn(`Error parsing annotation in PackageManifest ${pkg.metadata.name}`),
+              );
+            },
+          );
+          const filteredInfraFeatures = _.chain(parsedInfraFeatures)
```

Comment:
We should check whether `_.chain` pulls in all of lodash into our dependencies bundle or if tree shaking still works. Unfortunately, webpack-bundle-analyzer just broke in our repo for us to check. @vojtechszocs is working on a fix for that.

Reply:
The analyze script should be fixed in #8845 (thanks @vojtechszocs !)

Reply:
@spadgett would you prefer if I avoided chain for the time being and used `_.compact` and `_.map` directly until we sort out tree shaking? It looks like we already `import * as _ from 'lodash'` at the top of the file.

Reply:
@spadgett fwiw I ran `yarn run analyze` on:
## My branch + @vojtechszocs branch
<img width="1791" alt="Screen Shot 2021-05-03 at 3 29 17 PM" src="https://user-images.githubusercontent.com/21317056/116924258-9e07a280-ac25-11eb-86b1-f1107dd619b7.png">

## @vojtechszocs branch only

Reply:
Hey, @florkbr. Yeah, I think it would be better to avoid `_.chain` here, although admittedly I'm not sure if it is a problem or not based on the screenshots there. Do you mind pushing an update?

Reply:
@spadgett the screenshots show that only 1 new call to lodash was added (IE `chain`) resulting in 0.21 KB to be added to the bundle.

I went ahead and updated the logic anyway to avoid chain for now.